build/windows/ci-common.bat

```diff
@@ -1,13 +1,21 @@
 if defined KOKORO_BUILD_ID (
-    choco install llvm --version 6.0.1 -y
+    choco install cmake -y
 
     :: retry if choco install failed
     if errorlevel 1 (
-        choco install llvm --version 6.0.1 -y
+        choco install cmake -y
         if errorlevel 1 exit /b %errorlevel%
     )
+	
+	choco install llvm --version 8.0.0 -y
```

Comment:
nit: spacing

Reply:
Just realized my TextMate is set to use Tabs… :/

Reply:
😮